modules/analysis/templates/AOMDailyUsers.sh.erb

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+URL_PATTERN="https://addons.mozilla.org/en-US/firefox/addon/adblock-plus/statistics/usage-day-%s-%s.csv"
+NOW=$(date +"%Y%m%d")
+LOG_PATH=<%= @log_dir %>
+LOG_FILE="$LOG_PATH/AOMDailyUsers.log"
+
+if [ -f $LOG_FILE ]
+then
+    START=$(date --date=$(tail -1 $LOG_FILE | cut -c2-12 | tr "/" "-" )" +1 day" +"%Y%m%d")
+else
+    START=$(date --date=20120101 +"%Y%m%d")
+fi
+
+while [ $START -ne $NOW ]
+do
+    SOURCE="`printf \"$URL_PATTERN\" \"$START\" \"$START\"`"
+    wget -q -O- "$SOURCE" \
+	| tr -d '\r' \
+	| tail -1 \
+	| while IFS=, read date users; do
+	      DATE=$(date --date=$date +"%d/%b/%Y:%T %z")
+	      if [ $? -eq 0 ]; then
+		  printf "[$DATE] $users users\n" >> $LOG_FILE
```

Comment:
Please just write to `/dev/stdout`, and handle the $LOG_FILE within `cron {... }`.